Microsoft.Toolkit.Uwp.UI.Controls/PullToRefreshListView/PullToRefreshListView.cs

```diff
@@ -204,6 +198,41 @@ private void Scroller_DirectManipulationCompleted(object sender, object e)
 
         private void CompositionTarget_Rendering(object sender, object e)
         {
+            // if started navigating down, cancel the refresh
+            if (_scroller.VerticalOffset > 1)
+            {
+                CompositionTarget.Rendering -= CompositionTarget_Rendering;
+                _refreshIndicatorTransform.TranslateY = -_refreshIndicatorBorder.ActualHeight;
+                if (_contentTransform != null)
+                {
+                    _contentTransform.TranslateY = 0;
+                }
+
+                _refreshActivated = false;
+                _lastRefreshActivation = default(DateTime);
+
+                if (RefreshIndicatorContent == null)
+                {
+                    _defaultIndicatorContent.Text = "Pull to Refresh";
```

Comment:
That should be a feature in that case. Need to have a property for PullToRefreshLabel and ReleaseToRefreshLabel.